src/default_environment.rs

```diff
@@ -218,9 +240,24 @@ pub fn default_env() -> Env {
             let start = require_int_parameter("range", args, 0)?;
             let end = require_int_parameter("range", args, 1)?;
 
-            Ok(Value::List(
-                (start..end).map(|i| Value::Int(i)).collect::<List>(),
-            ))
+            cfg_if! {
+                if #[cfg(feature = "bigint")] {
+                    let mut i = start.clone();
+                    let mut res = Vec::with_capacity((end.clone() - start)
+                        .to_usize()
+                        .ok_or(RuntimeError::new("Failed converting `BigInt` to `usize`"))?
+                    );
+
+                    while i <= end {
+                        res.push(i.clone());
+                        i += 1;
+                    }
+
+                    Ok(Value::List(res.into_iter().map(Value::Int).collect::<List>()))
```

Comment:
It's weird that `BigInt` doesn't seem to implement `Step`. Still, I can't help but feel there might be a simpler way to do this? If not, this is probably fine

Reply:
Ah hmm, that's disappointing
https://github.com/rust-num/num-bigint/issues/68

Reply:
Yeah, I asked about it in Rust discord server, they gave me the same answer